pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -50,34 +68,39 @@ class Ametek7270(Instrument):
     ]
 
     sensitivity = Instrument.control(  # NOTE: only for IMODE = 1.
-        "SEN.", "SEN %d",
+        "SEN", "SEN %d",
         """ A floating point property that controls the sensitivity
         range in Volts, which can take discrete values from 2 nV to
         1 V. This property can be set. """,
         validator=truncated_discrete_set,
         values=SENSITIVITIES,
-        map_values=True
+        map_values=True,
+        check_set_errors=True,
+        dynamic=True,
+
     )
+
     slope = Instrument.control(
         "SLOPE", "SLOPE %d",
         """ A integer property that controls the filter slope in
         dB/octave, which can take the values 6, 12, 18, or 24 dB/octave.
         This property can be set. """,
         validator=truncated_discrete_set,
         values=[6, 12, 18, 24],
-        map_values=True
+        map_values=True,
+        check_set_errors=True

```

Comment:
You see, that the line `map_values` got changed, because you add another line, therefore it is good practice, to end every line, even the last one, with a comma:
```suggestion
        map_values=True,
        check_set_errors=True,
```

Reply:
done in next commit